app/views/posts/index.html.erb

```diff
@@ -1,25 +1,37 @@
-<div class = "container">
-<div class = "img">
-<img src="<%= @user.photo %>">
-</div>
-<div class="main">
-<h2> 
-<%= @user.name %>
-</h2>
-<p>Number of Posts: <%= @user.posts_counter %> </p>
-</div>
+<!-- index.html.erb -->
+<!-- index.html.erb -->
+<div class="container">
+  <div class="img">
+    <img src="<%= @user.photo %>">
+  </div>
+  <div class="main">
+    <h2><%= @user.name %></h2>
+    <!-- Update the variable name here -->
+    <p>Number of Posts: <%= @user.posts_counter %></p>
+  </div>
 </div>
+
+<!-- Add the placeholder here -->
+<ul class="photo">
+  <!-- Placeholder content goes here -->
+</ul>
+
 <ul class="posts">
-<% @user.posts.each do |post| %>
+  <% @posts.includes(:author, :comments).each do |post| %>
```

Comment:
[OPTIONAL] I can see you using posts here. But I can't see any @posts variable in the index of your users controller